backend/apps/core/utils/validators.py

```
@@ -0,0 +1,170 @@
+    """
+    if not phone_number:
+        return phone_number
+
+    # Normalize first
+    normalized = normalize_phone_number(phone_number)
+
+    # Validate format: +[1-3 digit country code][7-14 digit number]
+    pattern = r'^\+\d{1,3}\d{7,14}$'
+    if not re.match(pattern, normalized):
+        raise ValidationError(
+            f"Phone number must be in format +[country code][number] "
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)